fe/fe-core/src/main/java/com/starrocks/server/NodeMgr.java

```diff
@@ -741,9 +741,13 @@ public void addFrontend(FrontendNodeType role, String host, int editLogPort) thr
             throw new DdlException("Failed to acquire globalStateMgr lock. Try again");
         }
         try {
-            Frontend fe = getFeByHost(host);
-            if (null != fe) {
-                throw new DdlException("frontend use host [" + host + "] already exists ");
+            try {
+                if (checkFeExistByIpOrFqdn(host)) {
+                    throw new DdlException("FE with the same host: " + host + " already exists");
+                }
+            } catch (UnknownHostException e) {
```

Comment:
If throwing the exception, how to add the new POD?

Reply:
Adding POD is not effected by the `Add Follower/Observer ` command, it is controlled by k8s manager. The exception must be thrown if the specified host already in the cluster.